### ISSUE TITLE
Tapping into RichText doesn't move the cursor properly on Android

### ISSUE THREAD

**Issue by daniloercoli** [wordpress-mobile/gutenberg-mobile#135]
It seems that there are problems with the way the tap event is intercepted by the main list / block bottom toolbar / RichText component, that does result in weird behavior when you try to move the cursor to another part of the text (into the same RichText item) or when you try to cut&paste content.

- I found out that long tapping into the field doesn't' show the Cut&Paste menu.
- Also double tapping over a word doesn't select it.
- If you tap to move the cursor to a different position, the cursor is not moved but the bottom toolbar just toggle its state.

https://cloudup.com/cT2GShQAyY8

Above is a video that show how the Cut&Paste acts on the current version on a Nexus 5X.

Edit: I've just noticed that the video does't show the taps. 🤕  So it's hard to follow my interactions.

**Comment by hypest** [comment]
Adding a small observation: super-quickly tapping on some other word seems to work, but not the "normal speed" tapping.

**Comment by mzorz** [comment]
Fixed by #135
